Source/WebKit/Shared/PrintInfo.serialization.in

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2020 Apple Inc. All rights reserved.
```

Comment:
Looks like a copy/paste error?

Reply:
The serialization specification is identical to the code in the header, that's why I just copied the copyright to match the corresponding header.